docs/source/user/api_change.rst

```diff
@@ -10,13 +10,69 @@ The line number corresponds to the resulting line number after all changes are i
 Thus, be sure to implement each in order so that subsequent line numbers are correct.
 
 
-OpenFAST v3.2.0 to OpenFAST `dev`
+OpenFAST v3.3.0 to OpenFAST `dev`
 ----------------------------------
 
 None
 
 
 
+OpenFAST v3.2.0 to OpenFAST v3.3.0
+----------------------------------
+
+============================================= ==== ================= ======================================================================================================================================================================================================
+Added in OpenFAST dev 
+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+Module                                        Line  Flag Name        Example Value
+============================================= ==== ================= ======================================================================================================================================================================================================
+AeroDyn driver                                54\* WrVTK_Type        1           WrVTK_Type         - VTK visualization data type: (switch) {1=surfaces; 2=lines; 3=both}
```

Comment:
Seems like all of the new AeroDyn buoyancy changes are missing from this table.  And the AeroDyn user documentation has not been updated for buoyancy.

Reply:
Those changes are not included in this version. It turns out I have a mistake in the script that autogenerates the release notes. It searches for pull requests by date rather than by branch. Typically, this is ok, but in this case we are not merging `dev` so it didn't work. In any case, I've removed the buoyancy pull request from the release notes as well as the others that are in `dev` but not this release branch.